insights/parsers/bond.py

```diff
@@ -103,41 +103,71 @@ def parse_content(self, content):
         self._slave_speed = []
         self._slave_duplex = []
         self._primary_slave = None
+        self._up_delay = None
+        self._down_delay = None
+        self.name_slave = None
```

Comment:
@vishwanathjadhav please add a docstring for this public attribute.

Reply:
@bfahr - I have converted the public attribute to local variable.